apps/web/src/pages/Skills/ViewSkillPage.tsx

```diff
@@ -0,0 +1,279 @@
+
+  const skillFamilies = skill.families ?? [];
+  return (
+    <>
+      <div
+        data-h2-display="base(flex)"
+        data-h2-justify-content="base(center) p-tablet(flex-start)"
+      >
+        <Heading
+          level="h2"
+          color="primary"
+          Icon={IdentificationIcon}
+          data-h2-margin="base(0, 0, x1.5, 0)"
+        >
+          {intl.formatMessage({
+            defaultMessage: "Skill information",
+            id: "08IbRz",
+            description: "Heading for the 'view a skill' form",
+          })}
+        </Heading>
+      </div>
+      <CardSectioned.Root>
+        <CardSectioned.Item
+          data-h2-display="base(grid)"
+          data-h2-grid-template-columns="p-tablet(repeat(2, 1fr)) "
+          data-h2-gap="base(x1)"
+        >
+          <FieldDisplay label={intl.formatMessage(adminMessages.nameEn)}>
+            {skill.name.en}
+          </FieldDisplay>
+          <FieldDisplay label={intl.formatMessage(adminMessages.nameFr)}>
+            {skill.name.fr}
+          </FieldDisplay>
+          <FieldDisplay label={intl.formatMessage(adminMessages.descriptionEn)}>
+            {skill.description?.en}
+          </FieldDisplay>
+          <FieldDisplay label={intl.formatMessage(adminMessages.descriptionFr)}>
+            {skill.description?.fr}
+          </FieldDisplay>
+          <FieldDisplay
+            label={intl.formatMessage({
+              defaultMessage: "Keywords (English)",
+              id: "FiylOa",
+              description:
+                "Label displayed on the skill form keywords field in English.",
+            })}
+          >
+            {skill.keywords?.en ??
+              intl.formatMessage(commonMessages.notProvided)}
+          </FieldDisplay>
+          <FieldDisplay
+            label={intl.formatMessage({
+              defaultMessage: "Keywords (French)",
+              id: "fOl4Ez",
+              description:
+                "Label displayed on the skill form keywords field in French.",
+            })}
+          >
+            {skill.keywords?.fr ??
+              intl.formatMessage(commonMessages.notProvided)}
+          </FieldDisplay>
+          <div data-h2-grid-column="p-tablet(span 2)">
+            <FieldDisplay label={intl.formatMessage(adminMessages.category)}>
+              <Chip color="secondary">
+                {getLocalizedName(skill.category.label, intl)}
+              </Chip>
+            </FieldDisplay>
+          </div>
+          <div data-h2-grid-column="p-tablet(span 2)">
+            <FieldDisplay
+              label={intl.formatMessage(adminMessages.skillFamilies)}
+            >
+              <Chips>
+                {skillFamilies.map((family) => (
+                  <Chip key={family.id} color="primary">
+                    {getLocalizedName(family.name, intl)}
+                  </Chip>
+                ))}
+              </Chips>
+            </FieldDisplay>
+          </div>
```

Comment:
We need a null value for skill families section
![image](https://github.com/user-attachments/assets/7c395a5a-f048-4077-a056-eecf9d5e8944)

Reply:
Thanks, good catch!
[null state for skill families](https://github.com/GCTC-NTGC/gc-digital-talent/pull/11877/commits/06df9c139a586d302f575b29d80a75cda59867b6)